src/list/components/item-fields.js

```diff
@@ -9,35 +9,40 @@ import React from "react";
 import CopyToClipboardButton from "../../widgets/copy-to-clipboard-button";
 import FieldText from "../../widgets/field-text";
 import Input from "../../widgets/input";
+import Button from "../../widgets/button";
 import LabelText from "../../widgets/label-text";
 import PasswordInput from "../../widgets/password-input";
 import PasswordText from "../../widgets/password-text";
 
 import styles from "./item-fields.css";
 
 const fieldsPropTypes = PropTypes.shape({
+  isPopup: PropTypes.bool,
   origin: PropTypes.string.isRequired,
   title: PropTypes.string.isRequired,
   username: PropTypes.string.isRequired,
   password: PropTypes.string.isRequired,
 });
 
-export function ItemFields({fields, onCopy}) {
+export function ItemFields({fields, onCopy, isPopup}) {
+  const originEl = isPopup ? (<h4 className={styles.popupOrigin}>{new URL(fields.origin).host}</h4>)
```

Comment:
nit: Could help to reformat this over a few lines and/or use a plain old if statement - e.g.:
```javascript
const originEl = isPopup ? (
  <h4 className={styles.popupOrigin}>{new URL(fields.origin).host}</h4>
) : (
  <a className={styles.originLink} href={fields.origin}>{fields.origin}</a>
);
```
This is where I kind of wish we had Rust's `let foo = if ... { } else { }`